### DISCUSSION TITLE
notebooks/fds/9

### DISCUSSION THREAD

**Discussion by giscus** [NoughtQ/notebook#11] [Announcements]
# notebooks/fds/9

https://note.noughtq.top/notebooks/fds/9

**Comment by RalphFH** [comment]
强连通的DAG
拓扑序这里强连通和无环是冲突的概念

**Reply by NoughtQ** [reply]
感谢指出！当时脑子抽了，现在也没仔细看 :dizzy_face:，这句话没什么意义，就直接删了
